statsmodels/discrete/discrete_model.py

```diff
@@ -2525,22 +2525,23 @@ def _score_nbin(self, params, Q=0):
         y = self.endog[:,None]
         mu = self.predict(params)[:,None]
         a1 = 1/alpha * mu**Q
+
+        digamma_part = (special.digamma(y + a1) -
+                        special.digamma(a1))
+
         if Q: # nb1
```

Comment:
Sorry, one more thing before I merge. I think we should replace this with if Q == 1: to mitigate possible problems in the future (e.g. `Q=2` is a possibility at some point)? Since this is not my area, I want to be careful...

Reply:
That sounds like an excellent point for its own issue.  Out of scope for this.

Reply:
Well sort of. This PR represents a change in existing behavior in the case that `Q > 1` (I don't know if that case is even theoretically possible).

Reply:
Forget it. Trying to fix bugs or technical debt here isn’t worth the effort.  Closing.